kci_test

```diff
@@ -92,6 +92,8 @@ class cmd_get_lab_info(Command):
             'url': lab.url,
             'devices': lab_api.devices,
         }
+        dir = os.path.dirname(args.lab_json)
+        os.makedirs(dir)
```

Comment:
Please first check that the directory exists, otherwise this throws an exception:
```
Traceback (most recent call last):
  File "./kci_test", line 207, in <module>
    status = args.func(test_configs, lab_configs, args)
  File "./kci_test", line 96, in __call__
    os.makedirs(dir)
  File "/usr/lib/python2.7/os.py", line 157, in makedirs
    mkdir(name, mode)
OSError: [Errno 17] File exists: '/home/buildslave/build-trigger/workspace/kernel-build-trigger/labs'
```
https://staging-ci.kernelci.org/job/kernel-build-trigger/432/console

Reply:
Or maybe this really should be done outside of kci_test, by creating the directory before calling kci_test...

Reply:
in Python >= 3.2, we can just `os.makedirs(path, exists_ok=True)`

Reply:
Maybe wait for #261 to get merged and then rebase?

Reply:
More robust, I'll just add an os.path.exists() check.

Reply:
Fair enough.